pkg/controllers/resources/csistoragecapacities/syncer.go

```diff
@@ -29,7 +30,8 @@ func New(ctx *synccontext.RegisterContext) (syncertypes.Object, error) {
 	}
 
 	return &csistoragecapacitySyncer{
-		Mapper: mapper,
+		Mapper:   mapper,
+		Importer: pro.NewImporter(mapper),
```

Comment:
I think this resource doesn't work with importer as the direction is reversed and this is fromHost basically